MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet var errorView: ErrorView?
```

Comment:
The same as for the MVP solution, I recommend `errorView` to have a public getter and a private setter

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView?
```

Reply:
I did this change, but this kept me wondering because it seems weird to change the error view visibility just for the tests, also the MVC solution uses the same logic so maybe changes should be made there too

Reply:
Hi @caioortu,
You are right that this is done for tests only and this solution is not ideal. On the other hand it helps to decouple the tests from the view hierarchy in the storyboard.   Programming is all about trade offs. If you and your team